hypothesis-python/docs/tutorial/settings.rst

```diff
@@ -0,0 +1,85 @@
+Settings
+========
+
+The |@settings| decorator lets you control how Hypothesis runs a particular test. Using it looks like this:
```

Comment:
I think we want to explain at a conceptual level why we even have settings + what kinds of things they control, before getting into the decorator.  I'd also prefer to lead with profiles, and then present the decorator form as 'for the rare cases where you have a particular test which needs different configuration, ...'

Reply:
+1 on the first half, but IMO the settings decorator is much more common than profiles. I think the most common use of settings is to tune max_examples, and that's one of the settings where setting a global value via a profile makes the least sense. I think the flow of "settings changes hypothesis for all tests, and here's how to apply it to a single one" makes as much sense as "settings changes hypothesis for one test, and here's how to apply it to all of them", so I'd prefer to lead with the most common usage